src/NzbDrone.Core/Extras/ExtraFiles/ExtraFile.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using NzbDrone.Core.Datastore;
+using NzbDrone.Core.Extras.Metadata;
+using NzbDrone.Core.Parser;
+
+namespace NzbDrone.Core.Extras.ExtraFiles
+{
+    public class ExtraFile : ModelBase
```

Comment:
I'm not a huge fan of `ExtraFile`, but I wanted to separate Metadata from this file type as Metadata has been claimed by the metadata for other clients (Kodi, WDTV, etc).